app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -332,6 +331,22 @@ private void sendForms(FormRecord[] records) {
         }
     }
 
+    private static void logSubmissionAttempt(FormRecord record) {
+        String attemptMesssage = String.format(
+                "Attempting to submit form with id %1$s and submission ordering number %2$s",

```

Comment:
Yea I actually think this is super useful for debugging because the idea is that when logging at this series of logs for any app, the submissions IDs should always be sequential, and if 1 is missing then we know that something went wrong with a form submission.